criu/pidfd.c

```diff
@@ -145,6 +145,19 @@ static int create_tmp_process(void)
 static int free_dead_pidfd(struct dead_pidfd *dead)
 {
 	int status;
+	sigset_t blockmask, oldmask;
+
+	/* Block SIGCHLD to prevent interfering from sigchld_handler()
```

Comment:
nit: We usually use a slightly different format for comments:
/*
 * Block SIGCHLD to prevent interfering from sigchld_handler()